Azure-Samples/MLOpsDatabricks
# Full lint on the code to remove high priority issues

## Purpose

* Full review on the code to remove potential issues. Based on pylint

## Does this introduce a breaking change?

```
[X] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[X] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Run the AzDO pipeline pointing to this branch

* Test the code
  * Make sure to create the folder `/dbfs/model` on your cluster if you're running this manually.

```
python3 aml_service/pipelines/train_pipeline.py --cluster-id "<cluster-id>" --workspace-name "<workspace-name>" --resource-group "<resource-group>" --subscription-id "<subscription-id>" --app-id "<app-id>" --app-secret "<app-secret>" --tenant-id "<tenant-id>" --experiment-folder "./aml_service/experiment" --databricks-workspace "<databricks-workspace>" --databricks-access-token "<databricks-pat>" --databricks-compute-name "<databricks-aml-compute-name>" --project-folder "./" --train-script-path "src/train/train.py" --model-dir "/dbfs/model" --model-name "AzDO"
```

## What to Check
Verify that the following are valid
* The train script runs successfully and the model gets a new version on AML Service
